static-site/src/components/Groups/Tiles/index.tsx

```diff
@@ -7,6 +7,7 @@ import { UserContext } from "../../../layouts/userDataWrapper";
 import { GroupTile } from "./types";
 import { Group } from "../types";
 import { ExpandableTiles } from "../../ExpandableTiles";
+import { InternalError } from "../../ListResources/errors";
```

Comment:
Hmm, I think this warrants a separate error boundary for `static-site/src/components/Groups/Tiles/index.tsx`.

Reply:
Added 0af3c3a8 + 96f3018d to address this. Not with a separate error boundary but by reusing the existing one.